Python/compile.c

```diff
@@ -1630,29 +1636,25 @@ compiler_mod(struct compiler *c, mod_ty mod)
     return co;
 }
 
-/* The test for LOCAL must come before the test for FREE in order to
-   handle classes where name is both local and free.  The local var is
-   a method and the free var is a free var referenced within a method.
-*/
```

Comment:
What was this comment referring to?

Reply:
Before _PySt_GetScope this function worked out the cases:

https://github.com/python/cpython/blob/2cb94aba122b86dcda87d437eb36a860d14393d5/Python/compile.c#L5315